profiles/Octaviya-setup.yaml

```diff
@@ -0,0 +1,280 @@
+
+# Gear settings
+gear:
+- :adjective: chain
+  :name: hauberk
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: true
+- :adjective: coarse
+  :name: cowl
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: coarse
+  :name: gloves
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: target
+  :name: shield
+  :is_leather: false
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: parry
+  :name: stick
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: true
+- :adjective: brass
+  :name: knuckles
+  :is_leather: false
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: elbow
+  :name: spikes
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: true
+- :adjective: light
+  :name: spear
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: false
+  :swappable: false
+  :tie_to:
+- :adjective: bronze
+  :name: club
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: false
+  :swappable: false
+  :tie_to:
+gear_sets:
+  standard:
+  - chain hauberk
+  - coarse cowl
+  - coarse gloves
+  - target shield
+  - parry stick
+  - brass knuckles
+  - elbow spikes
+
+# Combat settings
+buff_spells:
+#  Shield of Light:
+#    mana: 15
+#    cambrinth:
+#    - 10
+#    - 5
+  Minor Physical Protection:
+    mana: 0
+    cambrinth:
+    - 2
+    - 1
+#  Manifest Force:
+#    mana: 10
+#    cambrinth:
+#    - 10
+#    - 10
+#  Sanctify Pattern:
+#    mana: 10
+#    cambrinth:
+#    - 10
+#    - 10
+#    cast: cast augmentation
+#  Major Physical Protection:
+#    mana: 10
+#    cambrinth:
+#    - 10
+#    - 10
+  # Persistence of Mana:
+  #   focus: blue-lacquered inkwell
+#  Benediction:
+#    mana: 15
+#    cambrinth:
+#    - 10
+#    - 10
+  Centering:
+    mana: 0
+    cambrinth:
+    - 2
+    - 1
+  # Auspice:
+  #   mana: 5
+  #   cambrinth:
+  #   - 4
+#offensive_spells:
+#- name: Fists of Faenella
+#  cast_only_to_train: true
+#  mana: 2
+#  cambrinth:
+#  - 4
+#- name: Soul Sickness
+#  mana: 1
+#  cambrinth:
+#  - 4
+# - name: Hydra Hex
+#   mana: 6
+#   cast: cast malediction offense
+#   harmless: true
+dance_skill: Small Edged
+training_abilities:
+  Tactics: 60
+  Hunt: 80
+  App Quick: 60
+  Perc: 120
+  Pray: 610
+  Faenella: 1200
+weapon_training:
+  Brawling: ''
+  Small Edged: windsteel adze
+  Light Thrown: rock
+skinning:
+  skin: true
+  arrange_all: false
+  arrange_count: 2
+  tie_bundle: true
+use_weak_attacks: true
+loot_subtractions:
+- bolt
+- bolts
+- arrow
+- arrows
+- rock
+- rocks
+# For ;afk
+health_threshold: 75
+
+# Non-combat settings
+crossing_training:
+- Theurgy
+- Augmentation
+- Utility
+- Warding
+- Attunement
+- Perception
+- Appraisal
+- Athletics
+#- Sorcery
+- Locksmithing
+- First Aid
+- Outfitting
+- Scholarship
+- Outdoorsmanship
+- Mechanical Lore
+train_workorders:
+- Tailoring
+training_spells:
+  Augmentation:
+    abbrev: Centering
+    symbiosis: true
```

Comment:
This looks suspicious for a low circle Cleric. Do you have these spells and can use symbiosis?